internal/app/proxy_stage_test.go

```diff
@@ -32,8 +32,10 @@ type ProxyStage struct {
x9u45hDBOGotWL5pVyPcxUNbGoaUuewVuqjmRlYDGZMZGa7kAb99vS1g2bFucPt2LZHNA7xruuvEfp8tjpeAgpNw0kc1hkl0lvvtdDlV071KuT6AHt0yfyMFFos1ykqSeL8JdeKAL3GAUUMzKy7OkNioN9LYmyP3clfFGPg0MzktceYrqlyrgsK03zygvxXrGLduEXyLYdLeeJs9LukOSZnVPsU5aSUU3SnZnsBf3xpf7HJpKjn3p2c4XFvPTkYVuntWHvXKy0MyPhaSfFdFiKUqcEXPDGHCAOmV4j7VXd1dTib7QXLs4A1Mlggc1P7fUzlxpbVAoEYjdzdFnHzh9KRwyUcisVq6aMJAA17wbWUrhh7qsxTTXhdhTSvghHtWmCuGHtzgTDaweXph2TtopNNaxBDoggJeCG1MmV36s4pHBmsIIOek7HJcd1lYF2CxNRJKgf5EiyEfDXStzJsv9lqRXDR7PdNwtWEEIhIchzlwl4XL9bkVXFX5O3fPPnSrCtqkvXEvprtt7DFmVDHsrS1Kxr18bzMu8wgrUorzVEJWakswv2Xws8FOTEFMUpkJ3nYgbZhef3t1TqMrL9itq4Lv0brhJGDGjdI5hfrdAUjrR7kq9sqkfkolc5jAVRQj9bK0PqjjGWRPCzQp5PFrProyfnZAmb1pminQqcSVhpDWvBz1ERPK45jKjlG5QXUTAwW48kSqsyGXwV6q9SyfOqzdSyWDlwyMtEjmfn7Gi81u8kcCRPsyPFhJLISHHQavOzRf64uaVSEGALoClxLiKMDi0vn7sYcc4nvNcOxhqEhLym7Hat6z4CmsOqv99e82SSP90HXZizNMv3o7mh30jvFWM8lLbDYuBxkoAArXNIHuzYdza03NkzoBNdI6v9yjzBDWcskfwSLheEs3JRRbhEFjkj32vZASKnqVNafe6T2pifTTAKuJwmF1n2sRWjwvLEctgja2LU8w9ljZgFiXgaP6SDoR3tVszKsSXMEQJcS8ozXYepqB1u0qJvce9if9yTfUZUS95oWlYWBqUR2oKVGKQKU28T5kTVYojXaJayO4liaUtTL6yQOLOmKh9OMHkohWcv7YpMPreDxms1A89TC2iigAlSImeMr5NP04B85Hc6r1Tw9OOSn9TiDGQCROwd5bBttBo0pcYSjebxSrenW9DB3DMsAdbS7yu5WY04VFxPUhDBiIzecxC4na5VKAJBRILnw9cEVnupy9Xy1IMrvUJy171UZDrbjgqWoz2jjH90KSMHuEkWjI4VjPMv9Pd1TkYqRMIC4Bm6FgFUt7nzAYveRn481ngQetSIsmIto75Bo127o8zpJzJeH4yqsZetUPoTYIuEhdrmsYAcF5rcjgbqTQeMieL6mIFf9SS4zA7ASA6k7CGBcljMGc4AwJqgq6QNtejoEM4K8PjriWvGTwzqfbn2hMkmf0QnYjDFp7k5in1talOmqfpo0rNprWcKd3Rv5pj9zYE6Dvn9hjWmmbLUJZaI8efcrPq7xSiRfjMS0b6bWPu095SjQypqbkaZqijDK962U9j4tlQGUu45gbaa0EMDmoB6FeInpjVcUgrlOdktHAihKICxTJXcs6dUx19xpyeR6GM5wcpqSFLGwA5JDV2sd6RYSMnWlIotMWnzAb1mYCdNqRBWUMroPXplbtRvwSZScCCuDZWCMWn8IgQrwG9s0ZRYB3K8oxnwy5LynR0H31By5nGb7cvMAINPwJCfJh8ZO0Rsj0xDPZYisfn8wTBdFIQIgi08ZvGcSvJJykEUEONlD8UuGwwN23rhFy55aSgKIw5UMG0uIiCetDhZX7XJghrVAeYv9F0lCteLqJB4CQ52f4x5ebxVuxVAyVcn8jNVTB42iZo9VAvOHPdjNwXxy0g2fP91k33begZTm5vxHpBfgrbOZInew4PDMMliB8RlvwuQdEtwhAfQWzc9xLmN5rKZw4sNStkuiwerP8JhXcaqfdlxp8Dhk8288aKwCYGXABuhoVJ8FEI8Hrqa6hYWk6b9xV3iarQKu1yo39u1o2aAVTwvDTKmRU4nqscOpW7jMxC0WXekHqr554oZjsQy31cKj0Y6HxSbiXyVGRKJiRtmZFY4KAtEv4GqSJ3DyUSGOv8HvEDph2LY7cOA3dj3oyNxG3WysJ27rbbqf4xTuDNE1c4kWwlcD8X3wKFMDRRykxpgYrtHjZYXMxJHeUomD7R96njCIU3c31IaTviKnOeIc1PTQiPJEVXBV0kTkkkF3wLzAXjKCs2eoQ2vPMPoVqRTaakAJMp0vHih7IgnC3B0R0aSLSxko0GXEgXd32lU7WtwsPhJIc5U90WaUNS714fB2H3jrAtMVgM2vGfEHP1liuAKf7opYWIiM8uZjS3QVjZrTWDz5De93iFRWXz7dyhoaZIZjKZroDnjK4wNcHa2aYUzEXKEiC8eO9183FfwWGmu6bmleKqOEBOLsBn0GHvxLzO4yj3If5OY9IiBfIxjlrffNmxcQqLR40yhwMmMRDDHx2cLnZHOB1f1JHqbWczQaLrDub3XfgGr7PFuhHEQ2feSepebsmeIWZm3BbcTX6DKahquWiU28NGzctUjYYf0ltqC4KAFiFV861r5b9vMx6g36XaA9pVQtdfe9isF6FBXzNDpHNvQo8GOFiQeFiV6YGNOXNIGP1NTt6AFKnZCU2jWPmfFgRHpKYowULV7mwfO25T7x6VydfbvkgTmPaATKQTlvIo2AY2SJ38vOpoEEIMEHHJNYV9qzJ0ztg0gCkSHA7gisFmu1jmEXIoj6kQXerLAsG5S2AQUGCxgFwb03rdblbIxXR8MqwySP7Cket4ZeOx2CbnDDpCeNfBtMk9l6bzsAZlf7Wl6EtEipKCMQGClDxv1FWokGS23cMASqIfQ8l4szjJMlknkWA3eJ1sP5HxthIhpU2rQRWc3hBAJHJPWzwBwC2L3q4DWnk2OPUEvgsCcwR49RvyyPywKC89BeyA8zoM5OMTkltABepHYK"
```

Comment:
Consider using `strings.Repeat`, this is [what strings_test.go does](https://go.dev/src/strings/strings_test.go#L336):

```suggestion
}

var largeString = strings.Repeat("long_string123BBmmF8BYezrBhCROOCRJfeH5k69hMKXH77TSvwF5GHUZFnbh1dsZ3d90HeR0jUIOovJJVS508uI17djeLFFSb7", 30) // 30 * 100 = 3000

Reply:
Done